diff --git a/src/hw/usb-hid.c b/src/hw/usb-hid.c
index fa4d9a2..7d3561c 100644
--- a/src/hw/usb-hid.c
+++ b/src/hw/usb-hid.c
@@ -59,7 +59,7 @@ usb_kbd_setup(struct usbdevice_s *usbdev
         // XXX - this enables the first found keyboard (could be random)
         return -1;
 
-    if (epdesc->wMaxPacketSize != 8)
+    if (epdesc->wMaxPacketSize != 8 && epdesc->wMaxPacketSize != 10)
         return -1;
 
     // Enable "boot" protocol.
